java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -222,33 +222,9 @@ private String getNewIssueLink(
     boolean hasDefaultCountry = !defaultCountry.isEmpty() && defaultCountry != "ZZ";
     String issueTitle = "Validation issue with " + phoneNumber
         + (hasDefaultCountry ? " (" + defaultCountry + ")" : "");
-
-    // Issue template. This must be kept in sync with the template in
-    // https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md.
-    StringBuilder issueTemplate = new StringBuilder(
-        "Please read the \"guidelines for contributing\" (linked above) and fill "
-        + "in the template below.\n\n");
-    issueTemplate.append("Country/region affected (e.g., \"US\"): ")
-        .append(defaultCountry).append("\n\n");
-    issueTemplate.append("Example number(s) affected (\"+1 555 555-1234\"): ")
-        .append(phoneNumber).append("\n\n");
-    issueTemplate.append(
-        "The phone number range(s) to which the issue applies (\"+1 555 555-XXXX\"): \n\n");
-    issueTemplate.append(
-        "The type of the number(s) (\"fixed-line\", \"mobile\", \"short code\", etc.): \n\n");
-    issueTemplate.append(
-        "The cost, if applicable (\"toll-free\", \"premium rate\", \"shared cost\"): \n\n");
-    issueTemplate.append(
-        "Supporting evidence (for example, national numbering plan, announcement from mobile "
-        + "carrier, news article): **IMPORTANT - anything posted here is made public. "
-        + "Read the guidelines first!** \n\n");
-    issueTemplate.append("[link to demo]("
-        + getPermaLinkURL(phoneNumber, defaultCountry, geocodingLocale, true /* absoluteURL */)
-        + ")\n\n");
-    String newIssueLink = "https://github.com/googlei18n/libphonenumber/issues/new?title=";
+    String newIssueLink = "https://issuetracker.google.com/issues/new?component=192347&title=";
```

Comment:
Let's move this right above the method and make it a constant:
private static NEW_ISSUE_URL = ...